<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGraphicsView" name="view1">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="view2">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuopen">
    <property name="title">
     <string>open</string>
    </property>
    <addaction name="openImage1"/>
    <addaction name="openImage2"/>
   </widget>
   <widget class="QMenu" name="harrisOperation">
    <property name="title">
     <string>Harris</string>
    </property>
    <addaction name="harrisImage1"/>
    <addaction name="harrisImage2"/>
   </widget>
   <widget class="QMenu" name="siftOperation">
    <property name="title">
     <string>SIFT</string>
    </property>
   </widget>
   <widget class="QMenu" name="distanceOperation">
    <property name="title">
     <string>Distance</string>
    </property>
    <addaction name="actionNCC"/>
    <addaction name="actionSSD"/>
   </widget>
   <addaction name="menuopen"/>
   <addaction name="harrisOperation"/>
   <addaction name="siftOperation"/>
   <addaction name="distanceOperation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="openImage1">
   <property name="text">
    <string>image1</string>
   </property>
  </action>
  <action name="openImage2">
   <property name="text">
    <string>image2</string>
   </property>
  </action>
  <action name="harrisImage1">
   <property name="text">
    <string>Harris 1</string>
   </property>
  </action>
  <action name="harrisImage2">
   <property name="text">
    <string>Harris 2</string>
   </property>
  </action>
  <action name="actionNCC">
   <property name="text">
    <string>NCC</string>
   </property>
  </action>
  <action name="actionSSD">
   <property name="text">
    <string>SSD</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
